### DISCUSSION TITLE
decrypt the database file

### DISCUSSION THREAD

**Discussion by samehmakin** [davidmartos96/sqflite_sqlcipher#72] [Q&A]
Hi,
for troubleshooting purposes I want to decrypt the database file after extracting it from an android device. I have the password that has been used to encrypt the database and I can open it on the device. but I want to open it on a windows machine as well.
is that possible?
thank you.

**Comment by davidmartos96** [comment]
I'd recommend SQLite Browser. It's a great app to inspect databases and supports opening password protected databases.
https://sqlitebrowser.org/dl/
